torch/_inductor/select_algorithm.py

```diff
@@ -714,6 +713,8 @@ def __call__(
         # generating a random torch.Tensor for benchmarking.
         input_gen_fns: Optional[Dict[int, Callable[[ir.Buffer], torch.Tensor]]] = None,
     ):
+        from .codegen.cuda.cuda_kernel import CUDATemplateCaller
```

Comment:
The explicit imports in `kernel/__init__.py` above changed the order of module loading and exposed a circular import between `select_algorithm.py` and `cuda_kernel.py`.